src/js/components/FormField/FormField.js

```diff
@@ -502,7 +507,26 @@ const FormField = forwardRef(
         }}
         onBlur={(event) => {
           setFocus(false);
-          if (contextOnBlur) contextOnBlur(event);
+
+          // if input has a drop and focus is within drop
+          // prevent onBlur validation from running until
+          // focus is no longer within the drop or input
+          if (
+            event.relatedTarget &&
+            focusWithinDropPortal(event.relatedTarget)
+          ) {
+            setFocusedDropElement(event.relatedTarget);
```

Comment:
It seems a bit odd to me that we are storing this in state. Can we just do the calculations each time this event handler is called?

Reply:
Fixed